cpp/include/copc-lib/copc/extents.hpp

```diff
@@ -0,0 +1,170 @@
+#ifndef COPCLIB_COPC_EXTENTS_H_
+#define COPCLIB_COPC_EXTENTS_H_
+
+#include <string>
+#include <vector>
+
+#include "copc-lib/las/utils.hpp"
+#include "copc-lib/las/vlr.hpp"
+
+namespace copc
+{
+
+class CopcExtent : public las::CopcExtentsVlr::CopcExtent
```

Comment:
For classes we inherit directly from external libs, let's inherit directly from them.
```suggestion
#include lazperf
class CopcExtent : public lazperf::CopcExtentsVlr::CopcExtent
```